.golangci.yaml

```diff
@@ -7,14 +7,19 @@ issues:
   max-same-issues: 0
   # Excluding configuration per-path, per-linter, per-text and per-source
   exclude-rules:
-    # exclude ineffassing linter for generated files for conversion
+    # exclude ineffassign linter for generated files for conversion
     - path: conversion\.go
       linters:
         - ineffassign
+    # limit the gocritic findings allowing for smaller prs
```

Comment:
what does this really means?

Reply:
I'll try to make this message clearer. The reason I excluded a lot of directories was to avoid the issue I had with another pr where there where a lot of different code owners that needed to approve and it needed frequent rebases. So by making this pr smaller I wanted to avoid that.

Reply:
oh, thanks, now it makes sense, 🙃